Src/IronPythonConsole/IronPythonConsole.csproj

```
@@ -9,6 +9,7 @@
     <CodeAnalysisRuleSet>..\..\IronPython.ruleset</CodeAnalysisRuleSet>
     <DocumentationFile>$(OutputPath)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <MacOS Condition="'$(OS)' == 'Unix' AND Exists('/Applications') AND Exists('/Library') AND Exists('/System') AND Exists('/Volumes')">true</MacOS>
```

Does this work? I have no mac to try it.
```suggestion
    <MacOS Condition="$([MSBuild]::IsOsPlatform('OSX'))">true</MacOS>
```

---

It works! I suppose we don't need `MacOS` anymore. Neither here nor in `Build.proj`.

---

Looks like the one in `Build.proj` is used by packaging.